<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scan_settings</class>
 <widget class="QDialog" name="scan_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>397</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scan Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="3" column="0">
      <widget class="QCheckBox" name="use_instrument_settings">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Use instrument settings</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QFrame" name="gridFrame_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>165</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="1">
         <widget class="QSpinBox" name="center_freq_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Number of Sweeps:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QRadioButton" name="center_span_radiobutton">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="toolTipDuration">
           <number>0</number>
          </property>
          <property name="text">
           <string>Center frequency and span</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Stop frequency:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="step_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string>MHz</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="stepType">
           <enum>QAbstractSpinBox::DefaultStepType</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QComboBox" name="no_sweeps_dropdown">
          <item>
           <property name="text">
            <string>MAXH</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QSpinBox" name="no_sweeps_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="4" colspan="3">
         <widget class="QRadioButton" name="start_stop_radiobutton">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Start and stop frequency</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Span:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="6">
         <widget class="QComboBox" name="start_freq_dropdown">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>GHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>kHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hz</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Step:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QComboBox" name="stop_freq_dropdown">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>GHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>kHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hz</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QCheckBox" name="checkBox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Log frequency</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Sweep points:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QSpinBox" name="stop_freq_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Center frequency:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="frequency_dropdown_center">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>GHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>kHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hz</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QSpinBox" name="start_freq_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="frequency_dropdown_span">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>GHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>kHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hz</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spanfreq_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99000</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="sweep_points_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="maximum">
           <number>99000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Start frequency:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QFrame" name="gridFrame_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>130</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="1">
         <widget class="QSpinBox" name="sweep_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string>[ms]</string>
          </property>
          <property name="maximum">
           <number>100000000</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="leveloffset_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string>[dB]</string>
          </property>
          <property name="maximum">
           <number>100000000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="attenuation_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string>[dB]</string>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="midLineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Reference level:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" alignment="Qt::AlignVCenter">
         <widget class="QSpinBox" name="referencelevel_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string>[dBm]</string>
          </property>
          <property name="maximum">
           <number>1000000000</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2" alignment="Qt::AlignLeft">
         <widget class="QCheckBox" name="referencelevel_checkbox">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="midLineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Level offset:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" alignment="Qt::AlignLeft">
         <widget class="QCheckBox" name="sweep_checkbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="midLineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Attenuation:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" alignment="Qt::AlignLeft">
         <widget class="QCheckBox" name="attenuation_checkbox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="midLineWidth">
           <number>1</number>
          </property>
          <property name="text">
           <string>Sweep:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" alignment="Qt::AlignLeft">
         <widget class="QCheckBox" name="leveloffset_checkbox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QFrame" name="gridFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>130</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QSpinBox" name="videoBW_spinbox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000000</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="videoBW_dropdown">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>GHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>kHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hz</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="resolutionBW_spinbox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100000000</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Video BW:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Use the same video bandwidth as resolution</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Resolution BW:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="resolutionBW_dropdown">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>GHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>kHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hz</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Detector:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Average</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EMI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Peak</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Units:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>dBm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dBuV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dBmV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dBuA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Volts</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Watts</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>scan_settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>scan_settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
